node/src/actors/chain_manager/mining.rs

```diff
@@ -836,6 +953,34 @@ fn build_block(
     (block_header, txns)
 }
 
+fn build_superblock(
+    block_headers: &[BlockHeader],
+    ars_identities: &[PublicKeyHash],
+    index: u32,
+    previous_hash: Hash,
```

Comment:
I like the new name, and a comment or an example would be nice

```suggestion
    last_block_in_previous_superblock: Hash,
```